NOTICES

```diff
@@ -755,26 +755,26 @@ SOFTWARE.
 
 The following software may be included in this product: @mapbox/tiny-sdf. A copy of the source code may be downloaded from git+https://github.com/mapbox/tiny-sdf.git. This software contains the following license and notice below:
 
-Copyright © 2016-2021 Mapbox, Inc.
-This code available under the terms of the BSD 2-Clause license.
+BSD-2-Clause
+Copyright (c) 2016-2022 Mapbox, Inc.
```

Comment:
out of curiosity, do these changes get auto-genned?

Reply:
Yes! There is a PR check that ensures it's up to date and if not, will tell you to run [make notices](https://github.com/streamlit/streamlit/blob/develop/Makefile#L373)